plopfile.mjs

```diff
@@ -0,0 +1,56 @@
+import path from "node:path";
+import * as glob from "glob";
+import autocompletePrompt from "inquirer-autocomplete-prompt";
+
+const PACKAGES_NEXTJS = "./packages/nextjs";
+
+/**
+ * @param {import("plop").NodePlopAPI} plop
+ */
+export default function(plop) {
+  plop.setPrompt("autocomplete", autocompletePrompt);
+  plop.setGenerator("stories", {
+    description: "Generates a Storybook stories file for a component",
+    prompts: [
+      {
+        name: "COMPONENT",
+        message: "Create a Stories file for which component? (you can search with wildcards)",
+        type: "autocomplete",
+        loop: false,
+        async source(answers, searchTerms) {
+          return glob.sync(`components/**/${searchTerms || ""}*`, {
+            cwd: PACKAGES_NEXTJS,
+            nodir: true,
+            nocase: true
+          }).map(file => {
+            const COMPONENT = {
+              FILE: file,
+              DIRNAME: path.dirname(file),
+              BASENAME: path.basename(file, path.extname(file)),
+            };
+            return {
+              name: file,
+              value: COMPONENT,
+            };
+          });
+        },
+        validate(input, _answers) {
+          const COMPONENT = input.value;
+          if (['.stories', '.test'].some(end => COMPONENT.BASENAME.endsWith(end))) {
```

Comment:
⛏️ (non-blocking) I wonder if this might be more robust:
```suggestion
          if (['.stories.', '.test.'].some(end => COMPONENT.BASENAME.endsWith(end))) {
```

Reply:
Since `BASENAME` already removes the `.ts / .tsx`, the current implementation should be solid